tests/test_google_storage.py

```diff
@@ -60,7 +72,10 @@ def test_save_to_file_downloads_blob_to_file_object(self):
         self.mock_bucket.blob.assert_called_once_with("path/filename")
         self.mock_blob.download_to_file.assert_called_once_with(mock_file)
 
-    def test_load_from_filename_uploads_blob_from_file_location(self):
+    def test_save_to_file_raises_exception_when_blob_does_not_exist(self) -> None:
+        pass
```

Comment:
Empty test, although, see my previous note about this not actually being possible in production.